### ISSUE TITLE
warning  libmamba Cache file "xxx.json" was modified by another program

### ISSUE THREAD

**Issue by sweettyler** [mamba-org/mamba#2456]
### Troubleshooting docs

- [X] My problem is not solved in the Troubleshooting docs

### How did you install Mamba?

Mambaforge

### Search tried in issue tracker

libmamba Cache file

### Latest version of Mamba

- [X] My problem is not solved with the latest version

### Tried in Conda?

Not applicable

### Describe your issue

It's quite easy to to duplicate:
1) run "mamba clean --yes --all";
2) run "mamba update --all" and you see there is NOT such a warning message;
3) run "mamba search python";
4) run again "mamba update --all" and you get the warnings:
  warning  libmamba Cache file "xxx.json" was modified by another program
  warning  libmamba Cache file "yyy.json" was modified by another program
  
  So in conclusion it is command "mamba search" cause these warnings

### mamba info / micromamba info

```shell
__    __    __    __
                 /  \  /  \  /  \  /  \
                /    \/    \/    \/    \
███████████████/  /██/  /██/  /██/  /████████████████████████
              /  / \   / \   / \   / \  \____
             /  /   \_/   \_/   \_/   \    o \__,
            / _/                       \_____/  `
            |/
        ███╗   ███╗ █████╗ ███╗   ███╗██████╗  █████╗
        ████╗ ████║██╔══██╗████╗ ████║██╔══██╗██╔══██╗
        ██╔████╔██║███████║██╔████╔██║██████╔╝███████║
        ██║╚██╔╝██║██╔══██║██║╚██╔╝██║██╔══██╗██╔══██║
        ██║ ╚═╝ ██║██║  ██║██║ ╚═╝ ██║██████╔╝██║  ██║
        ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝     ╚═╝╚═════╝ ╚═╝  ╚═╝

        mamba (1.4.2) supported by @QuantStack

        GitHub:  https://github.com/mamba-org/mamba
        Twitter: https://twitter.com/QuantStack

█████████████████████████████████████████████████████████████


     active environment : base
    active env location : /private/xxxx/mambaforge
            shell level : 1
       user config file : /private/xxxx/.condarc
 populated config files : /private/xxxx/mambaforge/.condarc
          conda version : 23.3.1
    conda-build version : not installed
         python version : 3.10.10.final.0
       virtual packages : __archspec=1=x86_64
                          __glibc=2.17=0
                          __linux=3.10.0=0
                          __unix=0=0
       base environment : /private/xxxx/mambaforge  (writable)
      conda av data dir : /private/xxxx/mambaforge/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /private/xxxx/mambaforge/pkgs
                          /private/xxxx/.conda/pkgs
       envs directories : /private/xxxx/mambaforge/envs
                          /private/xxxx/.conda/envs
               platform : linux-64
             user-agent : conda/23.3.1 requests/2.28.2 CPython/3.10.10 Linux/3.10.0-1160.83.1.el7.x86_64 centos/7.9.2009 glibc/2.17
                UID:GID : 26491:26491
             netrc file : None
           offline mode : False
```

**Comment by nevdelap** [comment]
I have the same symptoms, but in addition I get this in the` mamba update --all` output.

```
/home/nevd/miniconda3/lib/python3.10/site-packages/conda_package_streaming/package_streaming.py:19: UserWarning: zstandard could not be imported. Running without .conda support.
  warnings.warn("zstandard could not be imported. Running without .conda support.")
/home/nevd/miniconda3/lib/python3.10/site-packages/conda_package_handling/api.py:29: UserWarning: Install zstandard Python bindings for .conda support
  _warnings.warn("Install zstandard Python bindings for .conda support")
```

**Comment by leofang** [comment]
I hit this so often that I decided to google and found this issue. Would it be possible for mamba to relax this check in some way?

**Comment by aseedb** [comment]
Every single time I run mamba install I am getting this behavior

**Comment by chahank** [comment]
Same problem for me.

**Comment by zuiluo1116** [comment]
Same problem for me. Could be a mess up with conda?

**Comment by chahank** [comment]
Same for me.

**Comment by wuyh0328** [comment]
Have you solved it yet？

**Comment by jjerphan** [comment]
This issue has been resolved in mamba 2.0.
